src/infer_request.cc

```diff
@@ -82,6 +84,18 @@ InferRequest::ModelVersion()
   return model_version_;
 }
 
+bool
+InferRequest::SequenceStart()
```

Comment:
Existing Triton code just uses Flags(), so it would be good to continue that pattern. See sequence_batcher for example.